pkg/controller/plan/adapter/openstack/builder_test.go

```diff
@@ -16,3 +16,9 @@ var _ = Describe("vSphere builder", func() {
 		Entry("ubuntu 22", Ubuntu, "22.04.3", Ubuntu, "ubuntu"),
 	)
 })
+
+var _ = Describe("OpenStack Glance const test", func() {
+	It("GlanceSource should be glance, changing it may break the UI", func() {
+		Expect(GlanceSource).Should(Equal("glance"))
```

Comment:
```suggestion
		Expect(GlanceSource).Should(Equal(GlanceSource))
```

Reply:
isn't it always true?

Reply:
ah ok, I didn't pay much attention to the first part :)